core/src/main/scala/org/apache/spark/deploy/SparkSubmitArguments.scala

```diff
@@ -478,6 +482,8 @@ private[deploy] class SparkSubmitArguments(args: Seq[String], env: Map[String, S
       s"""
         |Options:
         |  --master MASTER_URL         spark://host:port, mesos://host:port, yarn, or local.
+        |  --insecure-skip-tls-verify  Ignore insecure certificates when connection with TLS to the
```

Comment:
It relates to the --master option. Would be strange to configure this in a property.